docs/configuration/pepr-policies.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Pepr Policies
+type: docs
+weight: 3
+---
+
+## Common Pepr Policies for UDS Core
+
+These policies are based on the [Kyverno](https://kyverno.io/policies/) policies.
+
+Exemptions can be specified by a [UDS Exemption CR](https://github.com/defenseunicorns/uds-core/blob/v0.27.0/src/pepr/operator/README.md). These take the place of Kyverno Exceptions.
+
+If a resource is exempted, it will be annotated as `uds-core.pepr.dev/uds-core-policies.<POLICY>: exempted`
+
+## Current Pepr Policies
+
+| Keyverno Policy | Exemption Reference | Description |
+| --------------- | :-----------------: | ----------- |
+| [Disallow Host Namespaces](https://kyverno.io/policies/pod-security/baseline/disallow-host-namespaces/disallow-host-namespaces/) | [`DisallowHostNamespaces`](https://github.com/defenseunicorns/uds-core/blob/v0.27.0/src/pepr/policies/networking.ts#L7-L35) | Subject:**Pod** <br> Severity:**high** <br><br> Host namespaces (Process ID namespace, Inter-Process Communication namespace, and network namespace) allow access to shared information and can be used to elevate privileges. Pods should not be allowed access to host namespaces. This policy ensures fields which make use of these host namespaces are set to `false`. |
```

Comment:
nit: could we add spaces after subject/severity on each of these? Ex:
```suggestion
| [Disallow Host Namespaces](https://kyverno.io/policies/pod-security/baseline/disallow-host-namespaces/disallow-host-namespaces/) | [`DisallowHostNamespaces`](https://github.com/defenseunicorns/uds-core/blob/v0.27.0/src/pepr/policies/networking.ts#L7-L35) | Subject: **Pod** <br> Severity: **high** <br><br> Host namespaces (Process ID namespace, Inter-Process Communication namespace, and network namespace) allow access to shared information and can be used to elevate privileges. Pods should not be allowed access to host namespaces. This policy ensures fields which make use of these host namespaces are set to `false`. |
```

Reply:
@mjnagel Yeah good point, its not super clear that you can click on the Exemption Reference column to go to the pepr mutation location. Not sure how to word that: `Exemption Reference and Pepr Mutation Link` is kinda long. 

- `Exemption Mutation Link`
-  `Pepr Mutation Link`
- `Pepr Exemption Link`
- `Exemption Pepr Link`

:shrug: :shrug:

Reply:
@mjnagel the table only contains the policies that are done. Policies that do not have mutations are not included

Reply:
Also somewhat hard to understand but the `Exemption Reference` column has links to the corresponding Pepr mutation code, unsure if i should rename that.

Reply:
Discussed offline - I think we want to explicitly call out the places where we have a `.Mutate` that does more than just add an exemption annotation (i.e. the places where we change `securityContext`). This main table should cover all of our `.Validate`s for policies, and then we can have a second table that covers any specific `.Mutate`s - these are actually what people tend to be caught off guard by more than just the policies.

Reply:
addressed these comments in the latest commit, i'll leave this thread open if there's more suggestions!